libraries.gradle

```diff
@@ -12,7 +12,7 @@ ext {
     junitVersion = '4.12'
 //    h2Version = '1.2.145'
     h2Version = '1.3.176'
-    bytemanVersion = '3.0.6'
+    bytemanVersion = '3.0.10'
```

Comment:
I would recommend using Byteman version `4.0.0-BETA5` : only version 4 is intended to work on Java 9.
I understand Byteman `3.0.10` seems to work, but that's pure luck ..

Reply:
I'm going to upgrade it